app/scripts/modules/openstack/image/image.reader.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.image.reader', [
+  require('exports?"restangular"!imports?_=lodash!restangular')
+])
+  .factory('openstackImageReader', function ($q, Restangular) {
+
+    function findImages(params) {
+      return Restangular.all('images/find').getList(params, {})
+        .then(function(results) {
+          return results;
+        })
+        .catch(function() {
+          return [];
+        });
+    }
+
+    function getImage(amiName, region, credentials) {
+      return Restangular.all('images').one(credentials).one(region).all(amiName).getList({provider: 'openstack'}).then(function(results) {
```

Comment:
It's inconsistent within this particular service, but consistent with the other `*ImageReader` service APIs. Not that it's "good" - it just hadn't been noticed or called out or fixed before.

I'm okay with it as-is. It might be nice (after this PR gets in) to remove the generic `imageReader` in the core module and refactor all the `*ImageReader` calls so that the provider doesn't need to be passed in.